huggle/generic.cpp

```diff
@@ -176,7 +176,7 @@ QString Generic::EvaluateWikiPageContents(ApiQuery *query, bool *failed, QString
     {
         if (reason) { *reason = EvaluatePageErrorReason_Unknown; }
         *failed = true;
-        return query->Result->ErrorMessage;
```

Comment:
This may work as a workaround but it's wrong, this shouldn't happen. There could be this code:
```
if (!query->Result)
    throw new Huggle::NullPointerException("query->Result", BOOST_CURRENT_FUNCTION);
```
 But replacing the error message just with "Unknown error", that is not a fix, just a workaround